### DISCUSSION TITLE
Shell Config Error, Connection Aborted

### DISCUSSION THREAD

**Discussion by kristiRule** [codefori/vscode-ibmi#1437] [General]
Having issues with my connection to IBM i. I get this error message. 

(I am relatively new to using this tool. It was working for me a few weeks ago with the local configs that were setup)

This extension can not work with the shell configured at [servername], since the output from shell commands have additional context. This can be caused by running commands like "echo" or other commands creating output in your shell start script. 

Connect aborted. 

We somewhat recently upgraded to version V7R4M0    L00

Thanks in advance for your guidance.

**Comment by worksofliam** [comment]
See this page which has some information regarding this error: https://halcyon-tech.github.io/docs/#/pages/tips/setup?id=error-in-shell-configuration
